### PR TITLE
[CI]Add model basic accuracy test(Qwen2-1.5B-Instruct)

### PR DESCRIPTION
**Pull request by hfadzxy**
### What this PR does / why we need it?
Add model basic accuracy test(Qwen2-1.5B-Instruct)
Currently, accuracy tests only run with the V0 Engine; V1 Engine support will be added in the 0.8.X release.

### CODE REVIEW

File: .github/workflows/vllm_ascend_test_main.yaml [hunk 1]
```diff
@@ -77,8 +77,8 @@ jobs:
 
       - name: Install vllm-project/vllm-ascend
         run: |
-          pip uninstall -y numpy
           pip install -r requirements-dev.txt
+          pip uninstall -y numpy
```
Comment (MengqingCao) [inline .github/workflows/vllm_ascend_test_main.yaml]: why do we need this change?

File: tests/entrypoints/test_accuracy.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+import os
+
+import lm_eval
+import pytest
+
+MODEL_NAME = "Qwen/Qwen2-1.5B-Instruct"
+TASK = "gsm8k"
+FILTER = "exact_match,strict-match"
+RTOL = 0.03
+EXPECTED_VALUE = 0.58
```
Comment (MengqingCao) [inline tests/entrypoints/test_accuracy.py]: please make some comments on why the rtol and expected value is set as this

File: tests/entrypoints/test_accuracy.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+import lm_eval
+import pytest
+
+MODEL_NAME = "Qwen/Qwen2-1.5B-Instruct"
```
Comment (MengqingCao) [inline tests/entrypoints/test_accuracy.py]: Just to make sure, why this is not qwen2.5?

Comment (MengqingCao) [reply tests/entrypoints/test_accuracy.py]: btw, I think we'd better add [meta-llama/Llama-3.2-1B-Instruct](https://huggingface.co/meta-llama/Llama-3.2-1B-Instruct) and [deepseek-ai/DeepSeek-V2-Lite](https://huggingface.co/deepseek-ai/DeepSeek-V2-Lite) in this ut, too.

File: tests/entrypoints/test_accuracy.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
+#
```
Comment (wangxiyuan) [inline tests/entrypoints/test_accuracy.py]: why created a new entrypoints folder?

File: tests/entrypoints/test_accuracy.py [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+import multiprocessing
+import os
+from multiprocessing import Queue
+
+import lm_eval
+import pytest
+import torch
+
+# pre-trained model path on Hugging Face.
+MODEL_NAME = "Qwen/Qwen2-0.5B-Instruct"
```
Comment (wangxiyuan) [inline tests/entrypoints/test_accuracy.py]: change to qwen2.5

File: tests/entrypoints/test_accuracy.py [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    queue.put(result)
+    del results
+    torch.npu.empty_cache()
+    gc.collect()
+    return
+
+
+@pytest.mark.skipif(
+    os.getenv('VLLM_USE_V1') == '1',
+    reason="V1 engine is fully supported in 0.8.X release, skipping this test."
```
Comment (wangxiyuan) [inline tests/entrypoints/test_accuracy.py]: why skip v1

**Review by wangxiyuan — APPROVED** [review]
